pydrive2/files.py

```diff
@@ -518,6 +518,36 @@ def Delete(self, param=None):
         """
         self._FilesDelete(param=param)
 
+    @LoadAuth
+    def Copy(self, target_folder, new_title = None, param=None):
+        """Copy this file to a new location.
+
+        :param target_folder: Folder where the file will be copied.
+        :type target_folder: GoogleDriveFile.
+        :param new_title: Name of the new file.
+        :type new_title: str.
+        :param param: addition parameters to pass
+        :type param: dict
+        """
+
+        if new_title is None:
+            new_title = self["title"]
+            
+        if param is None:
+            param = {}
+
+        param["fileId"] = self["id"]
+        param["supportsAllDrives"] = True
+        param["body"] = {}
+
+        param["body"]["parents"] = [{"id": target_folder["id"]}]
+        param["body"]["title"] = new_title
+
+        try:
+            self.auth.service.files().copy(**param).execute()
```

Comment:
btw, does this call return anything? new file for example? it would be handy to return it then from the function